docs/conf.py

```diff
@@ -33,8 +33,8 @@ def _compute_navigation_tree(context):
 
 # Pull in fix from https://github.com/sphinx-doc/sphinx/pull/11222/files to fix
 # "invalid signature for autoattribute ('ops.pebble::ServiceDict.backoff-delay')"
-import re
-import sphinx.ext.autodoc
+import re  # noqa: E402
+import sphinx.ext.autodoc  # noqa: E402
```

Comment:
Do you know why the tool is flagging [E402](https://www.flake8rules.com/rules/E401.html) - "multiple imports per line" - for these, when there only seems to be one per line?

Reply:
[E402](https://www.flake8rules.com/rules/E402.html) is "module level import not at top of file", the multiple imports per line one is E401.

I believe this wasn't picked up before because we didn't check anything in `docs`. It seems like it's reasonable to do so, and it's pretty small so I thought it could be included in this PR rather than broken out (but let me know if you disagree).

It seems possible that these could be moved up - I assume they're here because they are only used for the fix mentioned in the comment, but if the fix gets removed and we don't need them any more that would get picked up automatically. The PR that the code is from is still open, so it probably is still required.